tox.ini

```
@@ -84,18 +84,15 @@ exclude_lines =
   if TYPE_CHECKING:
   if debug:
 
-[pytest]
-script_launch_mode = subprocess
-
 [testenv:cli_func_v4]
 deps = -r{toxinidir}/requirements-docker.txt
 commands =
-  pytest --cov --cov-report xml tests/functional/cli {posargs}
+  pytest --script-launch-mode=subprocess --cov --cov-report xml tests/functional/cli {posargs}
 
 [testenv:py_func_v4]
 deps = -r{toxinidir}/requirements-docker.txt
 commands =
-  pytest --cov --cov-report xml tests/functional/api {posargs}
+  pytest --script-launch-mode=subprocess --cov --cov-report xml tests/functional/api --script-launch-mode-subprocess {posargs}
```

I think this is not needed here. I added this for the CLI tests initially with this arg as well (https://github.com/python-gitlab/python-gitlab/commit/79489c775141c4ddd1f7aecae90dae8061d541fe#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449L58)

```suggestion
  pytest --cov --cov-report xml tests/functional/api {posargs}
```

---

Done. Thanks.